app/js/components/manageApps/AddonList.jsx

```diff
@@ -9,45 +9,53 @@
  */
 import React from 'react';
 
-export const AddonList = ({handleDownload, install, appList, openPage, openModal}) => {
+export const AddonList = ({handleDownload, appList, openPage, openModal}) => {
   return (
     <tbody>
       {
         appList.map((app, key) => {
           return (
             <tr key={key}>
-              <td onClick={() => openPage(app)}>
+              <td onClick={() => openPage(app.appDetails)}>
```

Comment:
props shouldn't have arrow functions Using arrow functions or binding in JSX is a bad practice that hurts performance, because the function is recreated on each render and the previous function is garbage collected.

Reply:
@brayoh So theoretically, your argument has a good foundation(I used to have the same concerns too) but there has been no practical measure to show the actual degradation in performance caused by inline functions apart from theoretical suggestions. Premature optimization can actually make the app perform more slowly if there's no concrete measurement. Here is a good read that details your concern: https://cdb.reacttraining.com/react-inline-functions-and-performance-bdff784f5578